features/bootstrap/DoctrineContext.php

```diff
@@ -128,7 +122,7 @@ public function logout(): void
     }
 
     /**
-     * @BeforeScenario @createRegisterForm
+     * @BeforeScenario @create_register_form
```

Comment:
Keep the same case for all custom tags (camelcase or snakecase everywhere)
```suggestion
     * @BeforeScenario @createRegisterForm
```